ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Genesis/Governor.hs

```diff
@@ -354,7 +354,7 @@ densityDisconnect (GenesisWindow sgen) (SecurityParam k) states candidateSuffixe
   where
     densityBounds = do
       (peer, candidateSuffix) <- candidateSuffixes
-      let (clippedFragment, _) =
+      let (clippedFragmentWithTime, _) =
```

Comment:
(No action required in this PR.)

This `WithTime` suffix feels like a distraction, the GDD doesn't care about time at all. I think the "proper" thing to do here would be to make the code more abstract, ie consuming `AnchoredFragment hdr0`/`AnchoredFragment hdr1`, and doing the transformation in the tracer (via its `Contravariant` instance) instead.